Solutions/CohesitySecurity/Playbooks/Cohesity_Delete_Incident_Blobs/readme.md

```diff
@@ -0,0 +1,40 @@
+# Cohesity Delete incident blobs.
+## Summary
+This playbook deletes the blobs on Azure storage created by an incident that is generated by Cohesity function apps. For example, an automation rule can be created to delete the blobs used for an incident by running this playbook when the corresponding Sentinel ticket is closed.
+
+## Deployment instructions
+1. Click on the "Deploy to Azure" button to deploy the playbook. This step directs you to deploy an ARM Template wizard.
+[![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fcohesity%2FAzure-Sentinel%2FCohesitySecurity.internal%2FSolutions%2FCohesitySecurity%2FPlaybooks%2FCohesity_Delete_Incidient_Blobs%2Fazuredeploy.json)
+2. Fill in the required parameters:
+* __Playbook Name:__ Enter the playbook name here.
+
+## Post-Deployment instructions
+1. Make sure the user that runs the playbook has the role _Microsoft Sentinel Playbook Operator_ assigned. To assign the role,
+* Under the _Subscriptions_ tab from the _Home_ page, choose your subscription name.
+* Choose the _Access Control (IAM)_ option from the left pane.
+* Click on _Add > Add Role Assignment_ and add _Microsoft Sentinel Playbook Operator_ to the user.
+
+2. Authorize all connections
+* Go to _Logic Apps_ and choose your playbook
+* In the _Development Tools_ sections select _API Connections_. In the left pane you'll see the list of connections that you'll need to authorize
+  * Authorize the Azure blob storage connection by selecting it and clicking on _General\Edit API Connection_
+    * Enter your connection name, storage account and access key. You can find them by selecting your storage account [here](https://portal.azure.com/#view/HubsExtension/BrowseResource/resourceType/Microsoft.Storage%2FStorageAccounts) and then choosing _Security+networking\Access keys_).
+
+3. For the playbook to run, there is a need to assign the Microsoft Sentinel Responder role to the playbook's managed identity.
+* Under the _Subscriptions_ tab from the _Home_ page, choose your subscription name.
+* Choose the _Access Control (IAM)_ option from the left pane.
+* Click on _Add > Add Role Assignment_ and add _Microsoft Sentinel Responder_ managed identity role to the playbook.
+
+4. (Recommendation) You can create an automation rule to close the corresponding ServiceNow ticket when the corresponding Sentinel ticket is closed.
```

Comment:
You probably copied and pasted. There's no ServiceNow involved into this book.

Reply:
Done.